src/components/Task/Difficulty/DifficultyLabel/DifficultyLabel.tsx

```diff
@@ -6,67 +6,52 @@ import { DifficultyEnum } from '../../../../models/difficulty.enum';
 
 interface DifficultyLabelProps {
     difficulty: DifficultyEnum;
+    active?: boolean;
 }
 
 class DifficultyLabel extends React.Component<DifficultyLabelProps> {
     render() {
-        const { difficulty } = this.props;
+        const { difficulty, active } = this.props;
         const colors = colorData.difficultyColor;
         let color;
         if (difficulty in colors) {
             color = colorData.difficultyColor[difficulty];
         } else {
             color = colorData.difficultyColor.UNKNOWN;
         }
-        const { backgroundColor } = color;
         const { borderColor } = color;
-
+        let text;
         switch (difficulty) {
             case DifficultyEnum.EASY:
-                return (
-                    <span
-                        className="difficulty-label"
-                        style={{
-                            backgroundColor,
-                            borderColor,
-                        }}>
-                        {t.t('component.task.difficulty.easy', 'Easy')}
-                    </span>
-                );
+                text = t.t('component.task.difficulty.easy', 'Easy');
```

Comment:
Changing this is not nowending. just makes the code more confusing with too many variables